docker/scripts/create.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
```

Comment:
Reason for creating new script file to launch demo (we already have `demo-up.sh`) 

Maybe adding `.bat` scripts is a good idea, since we cannot be sure if a user on windows has gitbash/WSL

Reply:
my bad